### DISCUSSION TITLE
https://blog.3place.cn/3place/lmyn/LMYN022.html

### DISCUSSION THREAD

**Discussion by giscus** [hbaolong/hbaolong.github.io#68] [General]
# https://blog.3place.cn/3place/lmyn/LMYN022.html

https://blog.3place.cn/3place/lmyn/LMYN022.html

**Comment by picklecai** [comment]
说服十车王的，是婆私吒。
众友连哄带吓的长篇累牍，没起作用。或者说，只是把人吓昏。
而婆私吒说完，十车王就笑容满面、满怀愉悦地把孩子送出去了。

**Comment by picklecai** [comment]
两个人都带着箭壶，
他们的光辉照十方；
像是两条三头的蛇，
走在高贵众友的身旁。

这个比方，真是一言难尽。

**Comment by picklecai** [comment]
天鼓声声齐鸣，螺声鼓声争响，牟尼现在启程。 

锣鼓齐鸣地欢送。

**Comment by picklecai** [comment]
他们走了半由旬路，
来到了萨罗逾河的南岸，
众友仙人喊了声：“罗摩！”
说出了甜蜜的语言。 （1.21.9）

过了这条河，就真正辞别家乡了：
劝君更尽一杯酒，西出阳关无故人。
风萧萧兮易水寒，壮士一去兮不复还。
旦辞爷娘去，暮宿黄河边，不闻爷娘唤女声，但闻黄河流水鸣溅溅。旦辞黄河去，暮至黑山头，不闻爷娘唤女声，但闻燕山胡骑鸣啾啾。

过了这条河，就由少年儿童，转为大人了。以后的路，要自己走。

**Comment by picklecai** [comment]
不要让时间空过。
要学习这两个咒语：婆罗和阿底婆罗。
你将不会疲劳，不发烧，你的形貌也不会衰败。
不管你是睡着还是喝醉，恶魔们不会把你伤害。 （1.21.10）

众友开始行使承诺，带徒弟了。确实和乌巢禅师传心经比较像。

**Comment by picklecai** [comment]
在这个世界上，讲到英勇，
没有任何人媲美你的双臂。
在三个世界中，罗摩呀！
也没有任何人赶得上你。 （1.21.12）

讲到幸福，讲到和气，
讲到认识，讲到机智，
讲到回答问题，纯洁者！
你在世上，没人能比。 （1.21.13）

传完术之后，做心理疏导。

**Reply by picklecai** [reply]
心理建设，是指导员的角色。

**Comment by picklecai** [comment]
婆罗和阿底婆罗，是一切智慧的母亲。

**Comment by picklecai** [comment]
俱尸迦的那个儿子，
把当师傅的工作做完。
三个人愉快地过了一夜，
在萨罗逾河的岸边。 （1.21.19）

果然是师父，是导师。
即使是神话，天赋神力，也要让他有入门导师。这个理念不错。

**Comment by picklecai** [comment]
纯洁的罗摩满面生春，
他于是就沐浴在河水内。
接过了仙人的两个咒语，
他内心里得到了安慰。
罗摩得咒语欢欣鼓舞，
光彩焕发，精力充沛。 （1.21.18）

将要去闯世界的少年人：满面生春，欢欣鼓舞，光彩焕发，精力充沛。
